matisse/src/main/java/com/zhihu/matisse/engine/ImageEngine.java

```diff
@@ -33,10 +33,10 @@
      * @param context     Context
      * @param resize      Desired size of the origin image
      * @param placeholder Placeholder drawable when image is not loaded yet
-     * @param imageView   ImageView widget
+     * @param iv   ImageView widget
      * @param uri         Uri of the loaded image
      */
-    void loadThumbnail(Context context, int resize, Drawable placeholder, ImageView imageView, Uri uri);
+    void loadThumbnail(Context context, int resize, Drawable placeholder, ImageView iv, Uri uri);
```

Comment:
Well, such naming style can lead to sort of confusion, and we can wrap the line to avoid reaching the column limit.